Dockerfile.dev

```diff
@@ -0,0 +1,4 @@
+FROM node:alpine
```

Comment:
I don't have any idea about this docker stuff, what exactly it is, and where to learn this.

Reply:
This is a good video to understand the concept: https://youtu.be/Gjnup-PuquQ